modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,219 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
+  ingress:
+    enabled: true
+    #default value is nginx. It can be either nginx or haproxy
+    className: nginx
+    annotations: {}
+#      "demo": "value"
+#      "demo2": "value2"
+    tls:
+      enabled: false
+      config: []
+#      - secretName: "demo2"
+#        hosts:
+#          - localhost
+----
+
+==== Install the Helm chart
+
+The following steps assume that you have a Kubernetes cluster running and a standalone Neo4j helm chart installed.
+The standalone Neo4j has a Neo4j service with the name _standalone-admin_, and it has _:7474_ an _:7687_ opened.
+To verify that, run `get all, pvc, pv, configmaps, secrets`.
+
+. You need to have an Ingress controller for the Kubernetes Ingress to work.
+You can use the following command to install the Nginx Ingress controller:
++
+[source,shell]
+----
+helm upgrade --install ingress-nginx ingress-nginx \
+      --repo https://kubernetes.github.io/ingress-nginx \
+      --namespace ingress-nginx --create-namespace
+----
+
+. If your Neo4j is running on _:443_, then set the `tls.enabled` to `true` and create a secret with the following command:
+.. Create Ingress certificates:
++
+[source,shell]
+----
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ingress.key -out ingress.cert -subj "/CN=localhost/O=neo4j" -addext "subjectAltName = DNS:localhost"
+----
+.. Create Kubernetes secret using the Ingress certificates:
++
+[source,shell]
+----
+kubectl create secret tls ingress-cert --key /path/to/your/certs/ingress.key --cert /path/to/your/certs/ingress.cert
+----
+
+. Configure the _ingress.yaml_ file with the correct values for the `serviceName` and `secretName`.
+Ensure that the `secretName` is the same as the one created in the previous step.
+Enable TLS by setting `tls.enabled` to `true`.
++
+[source, yaml]
+----
+reverseProxy:
+    image: neo4j/neo4j-reverse-proxy:5.12.0
+    serviceName: "standalone-admin"
+    ingress:
+        enabled: true
+        tls:
+            enabled: true
+            config:
+                - secretName: ingress-cert
+                  hosts:
+                    - localhost
+----
+
+. If your Neo4j is running on _:80_, then set the `tls.enabled` to `false` and create the _ingress.yaml_ file with the following content:
++
+[source, yaml]
+----
+reverseProxy:
+    image: neo4j/neo4j-reverse-proxy:5.12.0
+    serviceName: "standalone-admin"
+    ingress:
+        enabled: true
+        tls:
+            enabled: false
+            config:
+                - secretName: ingress-cert
+                  hosts:
+                    - localhost
+----
+. Install the reverse proxy Helm chart using the following command:
++
+[source,shell]
+----
+helm install rp neo4j/neo4j-reverse-proxy -f /path/to/your/ingress.yaml
+----
+. Open the Neo4j browser on _https://localhost:443_ or _http://localhost:80_ and log in with your credentials.
```

Comment:
this is incorrect...you have to get the ingress ip using the below command 

```
kubectl get ingress/rp-reverseproxy-ingress -n default -o jsonpath='{.status.loadBalancer.ingress[0].ip}'"
```

and use the above ip instead of localhost

Reply:
I have not used localhost in the my video..not sure why its here

Reply:
I was confused because the host is localhost in the config. Also, I was wondering why we need to configure tls when it is disabled.